.github/workflows/release.yaml

```diff
@@ -80,14 +78,25 @@ jobs:
           # See e2e-test.yaml: the individual yamls are flattened inside the artifact.
           path: rendered_manifests
 
+      - name: rename vm-builder files
+        run: |
+          # FOR DEBUGGING ONLY:
+          ls *
+          # add the arch-specific suffix to the binary name, so they don't conflict when used as
+          # assets attached to the release.
+          mv vm-builder-amd64/vm-builder{,-amd64}
```

Comment:
what does "mv vm-builder-amd64/vm-builder{,-amd64}" do (without a second operand)?

Reply:
Could it be simplified to an expanded version? Doesn't look like it is much longer, but so much easier to read

Reply:
Fair enough, fixed in 1984ba7b82e6d02c6a999c230609a4fd7cdeb594